src/main/java/com/autotune/common/utils/CommonUtils.java

```diff
@@ -315,4 +335,164 @@ public static boolean isInvalidDataSource(DataSourceInfo datasource) {
         return datasource == null || datasource.getAuthenticationConfig() == null ||
                 datasource.getAuthenticationConfig().toString().isEmpty();
     }
+
+    public static void markAcceleratorDeviceStatusToContainer (ContainerData containerData,
```

Comment:
What is the reasoning for adding this to `autotune/common/utils`? In general, I would try to limit of scope of use for a method to only those that need it and not make too broad

Reply:
I thought to use it in services to check if there is GPU attached at the time of experiment creation so I have added it to common utils. Later the design is changed so I thought it's better to leave it in common so that sometime we might use it across the project to check for GPU usage and mark the containers.

Reply:
Lets increase the scope of the method when it is actually needed, I'd prefer to keep it local for now

Reply:
Sure, Can I move it to `RecommendationUtils`?

Reply:
Done @dinogun